core/src/main/java/com/exadel/etoolbox/backpack/core/servlets/model/PathModel.java

```diff
@@ -52,9 +56,17 @@ public String getPath() {
 
     /**
      * Gets the whether or not child pages excluded from a package build
```

Comment:
Replace "excluded" with "included" in the Javadoc. BTW, not just "included" but "are included")

Reply:
done